src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs

```diff
@@ -78,9 +89,7 @@ internal DtcProxyShimFactory(EventWaitHandle notificationEventHandle)
     {
         lock (_proxyInitLock)
         {
-#pragma warning disable IL2026 // This warning is left in the product so developers get an ILLink warning when trimming an app using this transaction support
```

Comment:
You can remove this entry now from https://github.com/dotnet/runtime/blob/992f178883f9ea79878125d64b33ef6a14a73fbd/src/libraries/System.Transactions.Local/src/ILLink/ILLink.Suppressions.LibraryBuild.xml#L4-L10

Reply:
Thanks @eerhardt, removed ILLink.Suppressions.LibraryBuild.xml.